resources/views/movies/index.blade.php

```diff
@@ -0,0 +1,26 @@
+@extends('layouts/app')
+
+@section('title', 'MovieDeck | Find the best movies!')
+
+@section('content')
+    <div class="container px-6 py-3 mx-auto md:flex">
+        <section class="text-gray-600 body-font">
+            <h1 class="font-medium text-gray-500 text-4xl px-5 pt-24">Popular Movies</h1>
+            <div class="container mx-auto flex px-5 py-24 md:flex-row flex-col items-center">
+                <div class="grid grid-flow-row sm:grid-cols-1 md:grid-cols-3 lg:grid-cols-5 gap-4 md:gap-28">
+                    @foreach($movies as $movie)
+                        <div>
+                            <a href="{{ url()->current() }}/{{ $movie->id }}">
```

Comment:
You can add a name route to your 'show' view 😄 , which could be named 'movie.show', then in here you would use the` route()` helper, so it would be` {{ route('movies.show', $movie) }}`.

Reply:
I didn't think of using the named route at all XD

Reply:
You can also change this route to use the helper 🥰